FirebaseSessions/Sources/SessionStartEvent.swift

```diff
@@ -78,6 +78,19 @@ class SessionStartEvent: NSObject, GDTCOREventDataObject {
     proto.session_data.data_collection_status.session_sampling_rate = samplingRate
   }
 
+  func set(subscriber: SessionsSubscriberName, isDataCollectionEnabled: Bool) {
```

Comment:
Should we remove the lines 67-70 since we have a separate setter function now? Also, where is this `set:` API called?

Reply:
We want to keep it in case the SDK isn't installed - but actually I need to handle that case and set them to NOT_INSTALLED